docs/Cloud_integrations/Azure_IoT/Azure_IoT_Hub/Configure_Azure_IoT_Hub_integration.md

```diff
@@ -47,3 +51,51 @@ Use the obtained credentials to establish the integration between {{ coiote_shor
       - click **Save** to keep the setting.
 
 Your integration should now be established. To get the integration up and running, try [synchronizing devices with Azure IoT Hub](Device_operations/Synchronize_devices_with_Azure_IoT_Hub.md)
+
+
+## Configuring Azure resources via CLI
+
+This section illustrates the configuration of Azure resources through the Command-Line Interface (CLI), offering a streamlined and scriptable approach to resource management. It is assumed that you have completed the following prerequisites:
+
+1. **Installed Azure CLI:**
+   Ensure that Azure CLI is installed on your system. If not, refer to the [official installation guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) for detailed instructions.
+
+2. **Authenticated to Azure CLI:**
+   Authenticate your Azure CLI by following the steps outlined in [this guide](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively).
+
+3. **Created Resource Group `coiote-dm-experiments`:**
+   Execute the following command to create the required resource group:
+   ```bash
+   # Adjust the group name as needed
+   az group create --name coiote-dm-experiments
+   ```
+
+### Resources creation and connection string retrieval
+
+Using this script, you can automate the creation of Azure resources. The script sanitizes and normalizes the username and then proceeds to create and configure a storage account and an IoT hub. Additionally, it retrieves and displays the connection strings associated with these resources, enabling a streamlined process for setting up essential Azure components.
+
+```bash
+# Sanitize and normalize the current user's username for Azure purposes
+export USER=$(echo $USER | sed 's/[^[:alnum:]]//g' | tr '[:upper:]' '[:lower:]')
+export GROUP=coiote-dm-experiments
+
+# Create and configure a storage account
+az storage account create -n ${USER}hub -g $GROUP
+az storage account show-connection-string -n ${USER}hub -g $GROUP
+
+# Create and configure an IoT hub
+az iot hub create -n $USER-hub -g $GROUP
+az iot hub connection-string show -n $USER-hub -g $GROUP
+```
+
+### Clean up resources
+
+To delete previously created Azure resources, including the specified storage account and IoT hub, execute this script: 
```

Comment:
add here some kind of warning that after the deletion integration stop working

Reply:
done